controllers/admin/AdminOrdersController.php

```diff
@@ -194,7 +194,7 @@ public static function setOrderCurrency($echo, $tr)
             $idCurrency = (int) $order->id_currency;
         }
 
-        return Tools::displayPrice($echo, $idCurrency);
+        return (new self())->context->getCurrentLocale()->formatPrice($echo, Currency::getCurrencyInstance($idCurrency)->iso_code);
```

Comment:
Mmm, `new self` seems like a bad idea, no to instanciate a controller to access the context, especially since controller may change the context in their init method (reminds me of a "bug" on shop context because `LegacyContext` juste made a `new AdminController()`)
If you are in a static context then it's better (actually less worse..) to call `Context::getContext()`

Reply:
Fixed